project/cmake/CMakeLists.txt

```diff
@@ -118,6 +118,12 @@ if(CORE_SYSTEM_NAME STREQUAL android)
   list(APPEND required_deps Zip)
 endif()
 
+# Missing mir support in these libraries
+if (ENABLE_MIR)
+  set(ENABLE_VAAPI OFF CACHE BOOL "Disabling VAAPI since no mir support" FORCE)
+  set(ENABLE_VDPAU OFF CACHE BOOL "Disabling VDPAU since no mir support" FORCE)
```

Comment:
please also add a message telling the user that HW acceleration is disabled because of mir